docs/sources/alerting/old-alerting/notifications.md

```diff
@@ -261,6 +262,18 @@ Alertmanager handles alerts sent by client applications such as Prometheus serve
 
 Grafana alert notifications can be sent to [Sensu](<(https://sensu.io)>) Go as events via the API. This operation requires an API key. For information on creating this key, refer to [Sensu Go documentation](https://docs.sensu.io/sensu-go/latest/operations/control-access/use-apikeys/#api-key-authentication).
 
+### WeCom
```

Comment:
With the changes suggested, we need to move this whole section to the `contact-points.md` file since we're not implementing WeCom in the old alerting just the new one.